configure.ac

```diff
@@ -1388,8 +1393,15 @@ fi
 AX_BOOST_SYSTEM
 AX_BOOST_FILESYSTEM
 
-dnl Opt-in to boost-process
-AS_IF([ test x$with_boost_process != x ], [ AX_BOOST_PROCESS ], [ ax_cv_boost_process=no ] )
+dnl Opt-in to Boost Process
+if test "x$boost_process" != xno; then
+AC_MSG_CHECKING(for Boost Process)
+AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <boost/process.hpp>]],
+ [[ boost::process::child* child = new boost::process::child; delete child; ]])],
+ [ AC_MSG_RESULT(yes); AC_DEFINE([HAVE_BOOST_PROCESS],,[define if Boost::Process is available])],
```

Comment:
Shouldn't the configure run fail when `--with-boost-process` is set but this test fails?

Reply:
Thanks, yes, this is now failing when Boost Process isn't available.

Reply:
Thanks. Yes this is better. I guess in the future when the option is changed to a semantic feature like `--enable-external-signer` it would make sense to mention in the error message what it's required for.
Not now though.